WoWsShipBuilder.Core/DataProvider/Updater/LocalDataUpdater.cs

```diff
@@ -351,18 +352,6 @@ private async Task ImageUpdate(IProgress<(int, string)> progressTracker, bool ca
                 progressTracker.Report((2, nameof(Translation.SplashScreen_ShipImages)));
                 await awsClient.DownloadImages(ImageType.Ship, fileSystem, versionName);
             }
-
-            // var camoImageDirectory = fileSystem.DirectoryInfo.FromDirectoryName(fileSystem.Path.Combine(imageBasePath, "Camos"));
```

Comment:
There is no current reason for camo, and i don't see a point in ever bothering with them, since they are purely visual. We should straight up delete this part of unused code

Reply:
this has already been removed